STMReco/src/SConscript

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
+#
+# Script to build the files found in this directory.
+#
+# Original author Rob Kutschke.
+#
+
+import os, re
+Import('env')
+Import('mu2e_helper')
+
+helper=mu2e_helper(env);
+rootlibs  = env['ROOTLIBS']
+
+rootlibs = env['ROOTLIBS']
+
+mainlib = helper.make_mainlib ( [ 'art_Persistency_Provenance',
+                                  'art_Persistency_Common',
+                                  'art_root_io_TFileService',
+                                  'art_Framework_Services_Registry',
+                                  'art_root_io_tfile_support',
+                                  'art_Utilities',
+                                  'canvas',
+                                  'cetlib',
+                                  'cetlib_except',
+                                  'CLHEP',
+                                  rootlibs
+                                ] )
+
+helper.make_plugins( [ mainlib,
+                       'mu2e_Mu2eUtilities',
```

Comment:
Have you stripped down both link lists to the minimal needed Mu2e libraries?

Reply:
I've stripped the link lists down to what I can in [8ec13f4](https://github.com/Mu2e/Offline/pull/856/commits/8ec13f455a3f6f81c572e3f581b8d2b59425018e)